plugin/gcp/firewall.py

```diff
@@ -0,0 +1,101 @@
+# #######
+# Copyright (c) 2014 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+from plugin.gcp.service import GoogleCloudPlatform
+from plugin.gcp.service import blocking
+
+
+class FirewallRule(GoogleCloudPlatform):
+    def __init__(self,
```

Comment:
I'd prefer init would only accept config and logger. other args (firewall and network or name and network) better be passed as args to specific methods (e.g. "create", "delete"), as required